src/index.html

```diff
@@ -22,6 +22,405 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a
+          class="logo__link"
+          href="index.html"
+        >
+          <img
+            class="logo__img"
+            src="./images/logo.png"
+            alt="logo_MOYO"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
```

Comment:
The class `is-active` should be modified as `nav__link--is-active` to follow the BEM modifier naming convention.